corehq/apps/reports_core/templates/reports_core/base_template_new.html

```diff
@@ -139,6 +138,7 @@
 {% endblock report_alerts %}
 {% block main_column_content %}
 <div class="report-controls">
+    {% if report.spec.description %}<h1><small>{{ report.spec.description }}</small></h1>{% endif %}
```

Comment:
`small` in an `h1` seems odd to me. seems like you could use `h2/3` and the `lead` class